modules/bitcoind.nix

```diff
@@ -195,6 +214,7 @@ in {
         chmod o-rw  '${cfg.dataDir}/bitcoin.conf'
         chown '${cfg.user}:${cfg.group}' '${cfg.dataDir}/bitcoin.conf'
         echo "rpcpassword=$(cat /secrets/bitcoin-rpcpassword)" >> '${cfg.dataDir}/bitcoin.conf'
+        chmod -R g+rX '${cfg.dataDir}/blocks'
```

Comment:
is that necessary despite setting sysperms?

Reply:
Yes. For blocks synced before sysperm was enabled.

Reply:
ok